Cargo.toml

```diff
@@ -29,6 +29,7 @@ fs-err = "2.5"
 glob = "0.3"
 home = "0.5"
 is_executable = "1"
+itertools = "0.10.1"
```

Comment:
This seems to be used only in tests. Could you move this to dev-dependencies?

Reply:
Sure. (Sorry, silly mistake.)